addons/project/static/src/components/project_right_side_panel/project_right_side_panel.js

```diff
@@ -57,6 +57,10 @@ export class ProjectRightSidePanel extends Component {
         };
     }
 
+    get showProjectUpdateSection() {
```

Comment:
This property is used for the profitability section so the name is not right. 🤔 

Moreover, I don't think we need that property, the helper could bei displayed when we have no data. 🤔